math/prime_test.py

```diff
@@ -15,4 +15,22 @@ def prime_test(n):
         if n%(j)==0 or n%(j+2)==0:
             return False
         j += 6
-    return True
\ No newline at end of file
+    return True
+
+
+def prime_test(n):
+    # prime numbers are greater than 1
+    if num > 1:
+    # check for factors
+        for i in range(2,num):
+           if (num % i) == 0:
+               print(num,"is not a prime number")
+               print(i,"times",num//i,"is",num)
+               break
+        else:
+           print(num,"is a prime number")
+        
+    # if input number is less than
+    # or equal to 1, it is not prime
+    else:
```

Comment:
comment out or remove the 
```python
else:
```
There's a syntax error and also ```else``` is not needed in the first place.

Reply:
Just fixed the syntax error, was able to run it smoothly. @goswami-rahul